not/localization/__init__.py

```diff
@@ -0,0 +1,98 @@
+"""The localization module provides localized user-facing strings
+
+generally, one need only import the polyglot singleton object
+
+This module uses an unorthodox approach to singletons to avoid keeping
+state in the module, which feels more awkward.
+
+When python 3.9 comes out in earnest, revisit using:
+https://www.python.org/dev/peps/pep-0603/
+"""
+
+import ctypes
+from json import load
+from locale import getdefaultlocale, normalize, windows_locale
+from os import getenv
+from pathlib import Path
+from platform import system
+from typing import Dict, Union
+
+DEFAULT_LOCALE = normalize("en")
+STRINGS_SUPPORTED = set("")
+
+
+class _Polyglot(Dict[str, str]):
+    """This class is responsible for lazy-loading the best-effort translation of user-facing
+    strings."""
+
+    language_alias_map = {
+        DEFAULT_LOCALE: {"en", "en_US"},
+    }
+    language_file_map = {
+        DEFAULT_LOCALE: "en.json",
+    }
```

Comment:
these might be better served as named tuples that contain both the Aliases and file maps, to avoid having to `normalize()` twice on other languages. DRY or something.